apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -32,6 +32,11 @@ type OpsRequestSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.clusterRef"
 	ClusterRef string `json:"clusterRef"`
 
+	// isCreateCluster indicates whether the OpsRequest is used to create a cluster.
+	// +kubebuilder:default=false
+	// +optional
+	IsCreateCluster bool `json:"isCreateCluster,omitempty"`
```

Comment:
I do not think a filed `IsCreateCluster` is necessary, is it enough to use `opsRequest.Spec.Type == appsv1alpha1.RestoreType` to check?